apisix/plugins/file-logger.lua

```diff
@@ -14,23 +14,23 @@
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
 --
-local log_util     =   require("apisix.utils.log-util")
-local core         =   require("apisix.core")
-local ngx          =   ngx
-local io_open      =   io.open
+local log_util = require("apisix.utils.log-util")
+local core = require("apisix.core")
+local ngx = ngx
+local io_open = io.open
 local is_apisix_or, process = pcall(require, "resty.apisix.process")
 
-
 local plugin_name = "file-logger"
-
+local std_out_file = "/dev/stdout"
```

Comment:
We should use Lua's writing to stdout feature, instead of Linux's special file system (which is broken in other OS).